src/main/java/com/thealgorithms/conversions/OctalToBinary.java

```diff
@@ -0,0 +1,39 @@
+package com.thealgorithms.conversions;
+
+import java.util.Scanner;
+public class OctalToBinary {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter an octal number: ");
+        int octalNumber = scanner.nextInt();
```

Comment:
This number is by default decimal right? (because java int works with decimal).
You need to get the number as a string.

Reply:
Check new PR